Duplicati/Library/Main/Operation/FilelistProcessor.cs

```diff
@@ -364,7 +418,15 @@ orderby n.Time descending
             }
 
             // cleanup deleted volumes in DB en block

```

Comment:
What's "DB en block" ?

Reply:
Not my comment, but it should be "en bloc" meaning "all at the same time"